packages/store/package.json

```diff
@@ -42,7 +42,8 @@
         "fast-check": "2.19.0",
         "jest-extended": "3.2.3",
         "memfs": "3.3.0",
-        "typescript": "4.3.5",
+        "jest": "27.5.1",
```

Comment:
Same comment as for the logger module: https://github.com/SAP/open-ux-tools/pull/708/files#r1113574214

Reply:
`"jest": ">=27.2.5",`
Seems to work better as the version from the root will be used and the warning is gone. 
Updated in 05539429acb5bcbd25dbe45d7fb97a177de7bb51